src/main/webapp/app/shared/layout/header/header-components.tsx

```diff
@@ -9,14 +9,14 @@ import appConfig from 'app/config/constants';
 
 export const BrandIcon = props => (
   <div {...props} className="brand-icon">
-    <img src="content/images/thirdcc_logo.png" alt="Logo" />
+    <img style={{ maxHeight: '100%' }} src="content/images/thirdcc_logo.png" alt="Logo" />
```

Comment:
Suggest to use bootstrap class "mh-100" if possible

Reply:
fixed at 07b3f4e3f3ebd838e261d28fbf7ce7dadb24db92